Tests/SwiftStdlibTests/KeyedDecodingContainerTests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  KeyedDecodingContainerTests.swift
+//  SwifterSwift
+//
+//  Created by Francesco Deliro on 23/10/2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+
+@testable import SwifterSwift
+
+private struct Video: Decodable {
+    let isPlaying: Bool
+    let isFullScreen: Bool?
+
+    enum CodingKeys: CodingKey {
```

Comment:
So can you get rid of this `enum`? Does it still work without it?

Reply:
Hi @guykogus , no I cannot remove the enum.

Reply:
Make it `private` at least, then :)

Reply:
Updated :)